platform/marvell-arm64/platform.conf

```diff
@@ -104,37 +193,31 @@ prepare_boot_menu() {
     fw_setenv ${FW_ARG} sonic_version_2 $sonic_version_2 > /dev/null
     BOOT1='echo " > Boot1: $sonic_version_1 - run sonic_image_1";echo;'
     BOOT2='echo " > Boot2: $sonic_version_2 - run sonic_image_2";echo;'
-    BOOT3='echo " > Boot3: ONIE - run onie-nand-boot";echo;'
+    if [ $PLATFORM_AC5X -eq 1 ]; then
+        BOOT3='echo " > Boot3: ONIE - run onie_emmc_boot";echo;'
+    else
+        BOOT3='echo " > Boot3: ONIE - run onie_nand_boot";echo;'
+    fi
```

Comment:
i do not know if this message really matters or not, suggest to change to run onie_boot and make it uniform for all platforms.